### ISSUE TITLE
Leaderboards

### ISSUE THREAD

**Issue by MetroidMan347** [itchio/itch.io#7]
I think it'd be really cool if there were Online Leaderboards for games where scores are really important. This is a pretty common thing on a lot of sites, so it'd be pretty sweet to see itch.io get it too.

**Comment by Bobbyjoness** [comment]
This doesn't really fit the scope

**Comment by Bobbyjoness** [comment]
Unless itch.io wants to become the indie varient of steam with all the bells and whistles

**Comment by Deozaan** [comment]
I agree with @Bobbyjoness. I personally don't need/want itch.io to have "social" features such as leaderboards or achievements.

**Comment by MetroidMan347** [comment]
@Deozaan That's an excellent point. I'd say the score table/achievement list shouldn't be public on the game page, but built merely as a backend for devs to access.

To just have a really easy way for games on the site to store such data.

**Comment by stolk** [comment]
Indies often cannot afford custom server infrastructure for leader boards and achievements.
itch.io would be the perfect candidate to fill the void in the market.
Please reconsider adding leader boards.

**Comment by Bobbyjoness** [comment]
stolk what do you mean? Digitalocean is 5 dollars a month. Sure a kid might not be able to afford that but any adult would be able to.

**Comment by quyse** [comment]
@Bobbyjoness still it's nice when you don't have to deal with servers and can just use API. It does not matter whether you're adult or kid. If it makes developer's like easier, why not? Also achievements and leaderboards may be integrated into itch.io (optionally shown on user's profile page, for example). I see you don't think that social features are good for itch.io - why? I would say it brings more fun, makes discovering new games easier and adds to games' commercial success. So itch.io shouldn't be just hosting site (as you can very well host on your own $5 server, integrate with Stripe and Paypal and sell your game yourself).

**Comment by bitbrain** [comment]
Any news on this?

**Comment by quazi-irfan** [comment]
It would be nice to have this feature.
